src/compiler/transformers/module/module.ts

```diff
@@ -892,24 +910,25 @@ namespace ts {
             // We don't need to check that the operator has SyntaxKind.plusplus or SyntaxKind.minusminus
             const operator = node.operator;
             const operand = node.operand;
-            if (isIdentifier(operand) && bindingNameExportSpecifiers) {
-                const sourceFile = getSourceFileOfNode(node);
-               const sourceFileName = sourceFile ? sourceFile.fileName : undefined;
-                if (bindingNameExportSpecifiers && bindingNameExportSpecifiers[sourceFileName] && hasProperty(bindingNameExportSpecifiers[sourceFileName], operand.text)) {
+            if (isIdentifier(operand) && bindingNameExportSpecifiersForFileMap) {
+                if (bindingNameExportSpecifiersForFileMap &&
+                    bindingNameExportSpecifiersForFileMap[currentSourceFileIdDuringSubstitution] &&

```

Comment:
As [above](https://github.com/Microsoft/TypeScript/pull/8991/commits/d5d2f8c19c69fb2da34019ce08cc987abe36b815#r66895073), create a local variable to hold this so we aren't performing the same lookup multiple times.